pkgs/applications/networking/cluster/ccoctl/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+, installShellFiles
+}:
+buildGoModule rec {
+  pname = "ccoctl";
+  version = "4.13.0";
+
+  src = fetchFromGitHub {
+    owner = "openshift";
+    repo = "cloud-credential-operator";
+    rev = version;
+    hash = "sha256-zWOIRfltCOvkYpk9flvS7pBtfeLhBGwAJNe6WfrEd9c=";
+  };
+
+  vendorSha256 = null;
```

Comment:
I assume this is because the dependencies are vendored in the repository?

Reply:
Yeah correct.

Reply:
Maybe you could add a short comment about it?